zerver/actions/realm_settings.py

```diff
@@ -309,7 +309,8 @@ def do_deactivate_realm(realm: Realm, *, acting_user: Optional[UserProfile]) ->
     realm.save(update_fields=["deactivated"])
 
     if settings.BILLING_ENABLED:
-        downgrade_now_without_creating_additional_invoices(realm)
+        billing_session = RealmBillingSession(user=acting_user, realm=realm)
```

Comment:
I think the bug in https://github.com/zulip/zulip/pull/27843/files#r1402968333 is that this should only have the `user` parameter passed, not also the `realm` parameter.